tests/TestCase/View/Widget/MultiCheckboxWidgetTest.php

```diff
@@ -369,6 +369,77 @@ public function testRenderTemplateVars()
         $this->assertHtml($expected, $result);
     }
 
+    /**
+     * Test label = false with checkboxWrapper option.
+     *
+     * @return void
+     */
+    public function testNoLabelWithCheckboxWrapperOption()
+    {
+        $data = [
+            'label' => false,
```

Comment:
Previously were there no labels?

Reply:
Before:
```html
<!-- First -->
<div class="checkbox">
  <input type="checkbox" name="test[]" value="1" id="test-1">
  <input type="checkbox" name="test[]" value="1" id="test-1">
</div>
<div class="checkbox">
  <input type="checkbox" name="test[]" value="2" id="test-2">
  <input type="checkbox" name="test[]" value="2" id="test-2">
</div>

<!-- Second -->
<div class="checkbox">
  <input type="checkbox" name="test[]" value="1" id="test-1">
</div>
<div class="checkbox">
  <input type="checkbox" name="test[]" value="2" id="test-2">
</div>
```

After:
```html
<!-- First -->
<div class="checkbox">
  <input type="checkbox" name="test[]" value="1" id="test-1">
  <label for="test-1">A</label>
</div>
<div class="checkbox">
  <input type="checkbox" name="test[]" value="2" id="test-2">
  <label for="test-2">B</label>
</div>

<!-- Second -->
<div class="checkbox">
  <input type="checkbox" name="test[]" value="1" id="test-1">
</div>
<div class="checkbox">
  <input type="checkbox" name="test[]" value="2" id="test-2">
</div>
```

So I think I fixed something wrong, but I am not sure the above HTML is an expected result or not. Now labels will appear even when label => false given, if checkboxWrapper contains {{input}}.

Reply:
The default template doesn't contain `{{input}}` so we should be ok.